client/my-sites/comments/comment-list/index.jsx

```diff
@@ -27,53 +34,96 @@ export class CommentList extends Component {
 
 	componentWillReceiveProps( nextProps ) {
 		if ( ! this.props.comments.length ) {
-			this.setState( { comments: nextProps.comments } );
+			this.setState( { comments: keyBy( nextProps.comments, 'ID' ) } );
 		}
 	}
 
+	showNotice = ( commentId, newStatus, previousStatus ) => {
+		const { translate } = this.props;
+
+		const [ type, message ] = get( {
+			approved: [ 'is-success', translate( 'Comment approved.' ) ],
+			unapproved: [ 'is-info', translate( 'Comment unapproved.' ) ],
```

Comment:
Hi! I've found a possible matching string that has already been translated **97** times:
`translate( 'Unapproved' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/actions-transitions-notices)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).